### DISCUSSION TITLE
implement sitemap

### DISCUSSION THREAD

**Discussion by ademsenel** [Dotnet-Boxed/Templates#61] [Q&A]
Hi Rehan 

Your project is perfect but i have a problem with my test project that i am new user of your template

I create one controller and i would like to implement sitemap but i didnt see my urls in the sitemap.xml file what is the problem also İ didn't know how i will implement could you explain how will it 

thank you

**Comment by RehanSaeed** [comment]
Hi, I'm not clear on what your problem is, can you give more information, even a code sample?

Adding items to the sitemap is as easy as adding the following code in SitemapService.cs:

```
var urlToControllerAction = urlHelper.AbsoluteRouteUrl("[Enter Controller Action Route Name Here]");
nodes.Add(
    new SitemapNode(urlToControllerAction)
    {
        Priority = 0.9
    });
```

**Comment by RehanSaeed** [comment]
I assume you've got things sorted. If not, re-open the issue.
